src/uct/sm/knem/knem_md.c

```diff
@@ -374,6 +372,12 @@ uct_knem_md_open(uct_component_t *component, const char *md_name,
                           ucs_status_string(status));
             }
         }
+
+        knem_md->reg_cost.overhead = md_config->rcache.overhead;
+        knem_md->reg_cost.growth   = 0; /* It's close enough to 0 */
```

Comment:
code duplication with lines 361-362

Reply:
you pointed me on the place where we are doing this initialization, thank you!